app/controllers/documents_controller.rb

```diff
@@ -39,6 +39,30 @@ def create
       else
         render action: 'new_folder'
       end
+    elsif params[:document][:file].kind_of?(Array)
```

Comment:
@Tr4pSt3R @seven1m Yeah gladly, I'm still in the process of learning ruby paradigms and I'd gladly take any help